test/exercise/fp/solution.rb

```diff
@@ -5,11 +5,14 @@ class << self
       # film["name"], film["rating_kinopoisk"], film["rating_imdb"],
       # film["genres"], film["year"], film["access_level"], film["country"]
       def rating(_array)
-        0
+        sorted_array = _array.reject { |film| film["rating_kinopoisk"].equal?(nil) }.reject { |film| film["rating_kinopoisk"].to_f == 0 }.select { |film| film["country"].to_s.include? "," }
```

Comment:
`.equal?(nil)` -> `.blank?`
`.reject { |film| film["rating_kinopoisk"].to_f == 0 }` what if _rating_ is _-100500_?
`film["country"].to_s.include? ","` what if _country_ is _nil_?